test/acceptance/mixins/logger.acceptance.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application, inject} from '@loopback/core';
+import {RestComponent, RestServer, get, param} from '@loopback/rest';
+import {sinon, Client, createClientForHandler} from '@loopback/testlab';
+import {LoggerMixin, Logger, LogArgs} from '../../..';
+
+describe('logger.mixin (acceptance)', () => {
+  // tslint:disable-next-line:no-any
+  let app: any;
+  let server: RestServer;
+  // tslint:disable-next-line:no-any
+  let spy: any;
```

Comment:
I am pretty sure sinon provides a type that can be used for spies, see e.g. how we are using `sinon.SinonStub` in [Thinking in LoopBack](https://github.com/strongloop/loopback.io/blob/ecb5e7e556e21e76f6dff1ccccb0d2e5a095bd5e/pages/en/lb4/Thinking-in-LoopBack.md#decouple-controller-from-repository).



```ts
let spy: sinon.SinonSpy;
```